WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -142,6 +142,8 @@ final class OrderListViewController: UIViewController {
         // and the toggle can be switch in the Settings section that resides in a different tab.
         viewModel.reloadSimplePaymentsExperimentalFeatureState()
 
+        viewModel.syncOrderStatuses()
+
         syncingCoordinator.resynchronize(reason: SyncReason.viewWillAppear.rawValue)
```

Comment:
nit: would be nice to group `viewModel.syncOrderStatuses` and `syncingCoordinator.resynchronize` in a separate function so that we know these two calls are needed for each sync

Reply:
It makes sense. Added to my todo list 👍